gradle.properties

```diff
@@ -2,7 +2,7 @@ fiatVersion=1.50.0
 korkVersion=7.243.0
 org.gradle.parallel=true
 spinnakerGradleVersion=8.32.1
-targetJava17=false
+targetJava17=true
```

Comment:
Is it also time to remove the java 11 docker images?

Reply:
Plus also, do we still need `-PenableCrossCompilerPlugin=true` in the gradle invocations?

Reply:
I reckon we go live with this small change in Igor, and if we're happy we come back and purge Java 11 from existence a few weeks after release.

Then for other repos just switch/purge.

Reply:
OK, let's do it. This'll make it into 1.36....